<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>291</width>
      <height>327</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="Reg">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Регистрация</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EnterName">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите название компании</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Name"/>
     </item>
     <item>
      <widget class="QLabel" name="EnterPassword">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Password1"/>
     </item>
     <item>
      <widget class="QLabel" name="Password1Error">
       <property name="font">
        <font>
         <italic>false</italic>
         <underline>false</underline>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="RepeatPassword">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
         <underline>false</underline>
         <strikeout>false</strikeout>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;повторите пароль&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>Повторите пароль</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Password2"/>
     </item>
     <item>
      <widget class="QLabel" name="Password2Error">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="EnterEmail">
       <property name="font">
        <font>
         <family>Georgia</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Введите электронную почту</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="Email"/>
     </item>
     <item>
      <widget class="QLabel" name="EmailError">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="RegistrateButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Зарегистрироваться</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
